R/validation.R

```diff
@@ -274,16 +274,16 @@ validate_outcomes_are_binary <- function(outcomes) {
   check <- check_outcomes_are_binary(outcomes)
 
   if (!check$ok) {
-    bad_cols <- glue::single_quote(check$bad_cols)
-    bad_msg <- glue::glue("{bad_cols}: {check$num_levels}")
-    bad_msg <- glue::glue_collapse(bad_msg, sep = "\n")
-
-    glubort(
-      "The outcome must be binary, ",
-      "but the following number of levels were found:",
-      "\n",
-      "{bad_msg}"
-    )
+    bad_col <- map(check$bad_cols, ~ cli::format_inline("{.val {.x}}"))
```

Comment:
Ok, now I'm in "don't repeat yourself" mode and think that a function should be considered.

Reply:
This one here is a slightly different pattern than the other 3 occurrences so I've preserved this one and moved the others into a little helper.